src/common/model/GenerationRequest.ts

```diff
@@ -34,5 +37,6 @@ export default interface GenerationRequest {
   dataCid?: string,
   carSize?: number,
   pieceCid?: string,
-  pieceSize?: number
+  pieceSize?: number,
+  filenameOverride?: string // when the car name is different from cid
```

Comment:
Who will set this value?

Reply:
After we implement the feature to allow user scan their own cars, we'll need another filename column I'm guessing.

Reply:
makes sens